test/language/expressions/greater-than-or-equal/bigint-and-incomparable-string.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Relational comparison of BigInt and string values
+esid: sec-abstract-relational-comparison
+features: [BigInt]
```

Comment:
The following info would be really helpful here:

```
7.2 StringToBigInt ( argument )

Apply the algorithm in 3.1.3.1 with the following changes:

- Replace the StrUnsignedDecimalLiteral production with DecimalDigits to not allow Infinity, decimal points, or exponents.
- If the MV is NaN, return NaN, otherwise return the BigInt which exactly corresponds to the MV, rather than rounding to a Number.
```

I can add that after merging this